README.md

```diff
@@ -5,6 +5,34 @@ Docker Images for RancherOS
 
 Run `make`
 
+## Details
+
+### Multi-arch Dockerfiles
+
+As a pre-cursor to native multi-arch support, we leverage some features of
+[dapper](https://github.com/rancher/dapper).
+
+In the Dockerfiles, you'll see lines like:
+
+```
+FROM rancher/os-centosconsole-base
+# FROM amd64=centos:7 arm64=skip arm=skip
+```
+
+The `rancher/os-centosconsole-base` does not actually exist. Dapper will download the
+arch specific image listed in the commented out `# FROM` line, and tag that, so the
+build can occur.
+
+## Special Entrypoint and CMD
+
+Console images also use special ENTRYPOINT and CMD settings, which are bind mounted
+into the container at run time:
+
+```
+ENTRYPOINT ["/usr/bin/ros", "entrypoint"]
+CMD ["/usr/sbin/console.sh"]
```

Comment:
This was switched to `ros console-init` and a `CMD` isn't specified in the image anymore. It's located in the service file instead (https://github.com/rancher/os-services/blob/master/d/debian.yml#L3). Maybe we can move these docs to a separate PR?